wasatch/AndorDevice.py

```diff
@@ -435,7 +435,7 @@ def update_config_from_eeprom(self, eeprom):
                 self.config_values[k] = v
 
     def _load_config_values(self):
```

Comment:
This function is messy. I recommend a docstring:
```
"""
Loads configuration from file `self.config_file` and populates `self.settings.eeprom` with members.
"""
```

This falls under the category of scope manipulation, which is bad, but if we're going to do it lets be clear about it in the comments.

Reply:
Documentation updated.

Reply:
Thanks for adding a docstring to `update_config_from_eeprom`.

I was actually referring to the function `_load_config_values`.
I wrote in that docstring as a separate PR: https://github.com/WasatchPhotonics/Wasatch.PY/pull/95/files

---

When a click to add a comment to a function, github only shows the first line above my comment:
![image](https://github.com/WasatchPhotonics/Wasatch.PY/assets/124081765/96019182-63af-4da6-8724-04ebc37fb105)